{{cookiecutter.project_shortname}}/ui/craco.config.js

```diff
@@ -8,6 +8,14 @@ module.exports = {
     mode: 'file',
   },
   webpack: {
+    configure: webpackConfig => {
```

Comment:
could you tell us more about this change on the next standup please? I would like to fully understand what is happening